src/containers/SignIn/SignIn.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { GoX } from 'react-icons/go';
+import signIn, { clearErrors } from '../../actions/SignIn';
+import styles from './SignIn.module.css';
+import SocialSignup from '../../components/SocialSignup';
+
+export class SignIn extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: "",
+      password: "",
+    }
+    this.signInTitle = "Welcome to Authors Haven";
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const { props } = this
+    props.dispatch(signIn(this.state));
```

Comment:
I think you can just have this to be `this.props.dispatch` directly

Reply:
When I try to use ```this.props.dispatch``` directly, Eslint throws the following error: ```[eslint] Must use destructuring props assignment (react/destructuring-assignment)```